### ISSUE TITLE
bootstrap script must Register External IPs and External Address

### ISSUE THREAD

**Issue by bzon** [awslabs/amazon-eks-ami#60]
The bootstrap script only registers Internal IPs and addresses.

**Comment by micahhausler** [comment]
This was an issue in upstream kubernetes, and is included in 1.11

* https://github.com/kubernetes/kubernetes/issues/63158
* https://github.com/kubernetes/kubernetes/pull/63170

**Comment by emman27** [comment]
Hi @micahhausler - I've taken a closer look at the eks bootstrap script. Seems that if I override the node IP using the `--node-ip` flag, it also overrides the primary ipv4 address. Here's a quick look at my userdata script that I'm using: 

```sh
EXTERNAL_IP="$(curl http://169.254.169.254/latest/meta-data/public-ipv4)"
INTERNAL_IP="$(curl http://169.254.169.254/latest/meta-data/local-ipv4)"

/etc/eks/bootstrap.sh ${cluster_name} --kubelet-extra-args "--node-ip=$INTERNAL_IP,$EXTERNAL_IP --node-labels=${kubelet_node_labels} --register-with-taints=${kubelet_node_taints}"
```

The script works fine as far as I'm aware of. What's interesting is the output of `kubectl get no -o yaml` later, which shows the correct external IP, correct external DNS, correct internal DNS, but wrong internal IP - a secondary private internal IP is being used. This results in the inability to use commands such as `kubectl exec` and `kubectl port-forward`, which basically means I can't use helm anymore (dependency on port-forwarding).
